scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -345,64 +345,72 @@
                                             (add-starting-pods this)
                                             (filter synthetic-pod->job-uuid))
             num-synthetic-pods (count outstanding-synthetic-pods)
-            {:keys [image user command max-pods-outstanding] :or {command "exit 0"}} synthetic-pods-config]
+            total-pods (-> @all-pods-atom keys count)
+            total-nodes (-> @current-nodes-atom keys count)
+            {:keys [image user command max-pods-outstanding max-total-pods max-total-nodes]
+             :or {command "exit 0" max-total-pods 1000000 max-total-nodes 1000000}} synthetic-pods-config]
```

Comment:
Why not have much lower defaults, maybe 32000 total pods and 1000 total nodes?

Reply:
Ok.